core/conversion/conversion.cpp

```diff
@@ -291,8 +298,141 @@ void EvaluateLoopBlock(ConversionCtx* ctx, const torch::jit::Node* n) {
     }
 }
 
+void ConvertLoopBlock(ConversionCtx* ctx, const torch::jit::Node* n) {
+    auto block = n->blocks()[0];
+
+    // max_trip_count and start_cond already evaluated
+    auto max_trip_count = ctx->evaluated_value_map[n->input(0)];
+    auto start_cond = ctx->evaluated_value_map[n->input(1)];
+
+    ctx->evaluated_value_map[block->inputs()[0]] = torch::jit::IValue(0);
+    auto trip_count = ctx->evaluated_value_map[block->inputs()[0]];
+
+    // map node inputs [recurrent values] -> node outputs [recurrent values]
+    MapIValues(ctx, n->inputs(), n->outputs(), 2, 0);
+
+    LOG_DEBUG(ctx->logger, "(Loop Evaluation) Evaluating loop " << *n);
```

Comment:
Can you change the prefix here to Loop Conversion so we can tell the difference?